Добавлена генерация архива с Эклипсом с плагинами

AlexChernov opened the pull request:
(aurusov/raox-deploy#1)

AlexChernov committed:
Добавлена генерация архива с Эклипсом с разными плагинами (aurusov/raox-deploy#1)

aurusov commented on assembly/pom.xml:
```diff
@@ -0,0 +1,219 @@
+								</artifactItem>
+								<artifactItem>
+									<groupId>ru.bmstu.rk9.raox.plugins.game5</groupId>
+									<artifactId>ru.bmstu.rk9.raox.plugins.game5</artifactId>
```
А где связанный пул-реквест на изменение имени пакета в плагине ?

AlexChernov commented:
https://github.com/aurusov/raox-game5/pull/67

aurusov commented:
ok

AlexChernov committed:
Create build-with-maven.md

AlexChernov committed:
Добавлена сборка архива с эклипсом под mac (aurusov/rdo-xtext#227)

AlexChernov commented:
@aurusov 2 коммита новых после ок
@bogachev-pa и/или @LeKaitoW посмотрите тоже, пожалуйста

aurusov commented:
Считаешь, что `${eclipse-subfolder}` лучше ?

bogachev-pa commented on .gitignore:
```diff
@@ -0,0 +1 @@
+assembly/target/*
```
Добавь ньюлайн

bogachev-pa commented:
Пока не очень могу понять, чем отличается `${eclipse-subfolder}` от `${eclipse-path}`? Первый - поддиректория второго? Можно пример?

AlexChernov commented:
`${eclipse-subfolder}` - Пустой string по-умолчанию
Сборка c параметром `-target-os=mac` запускает профиль `mac-os `, который меняет этот параметр на значение подпапки (`Contents/Eclipse`), в которой находятся файл `eclipse.ini` и папка `dropins`

AlexChernov committed:
Add newline

AlexChernov committed:
Rename properties:
${eclipse-subfolder} -> ${eclipse-content-relative-path}
${eclipse-path} -> ${eclipse-archive-root-path}

AlexChernov committed:
Update build-with-maven.md

add bash highlighting

aurusov commented:
ok

aurusov commented:
Хочется проверить, но не хватает ока. Паша ?

bogachev-pa commented:
`ok`
